src/processTree.ts

```diff
@@ -125,27 +125,25 @@ export function getProcesses(one: (pid: number, ppid: number, command: string, a
 
 		} else {	// linux
 
-			proc = spawn('/bin/ps', [ '-ax', '-o', 'pid,ppid,comm:20,command' ]);
-			proc.stdout?.setEncoding('utf8');
-			proc.stdout?.on('data', lines(line => {
-
-				const pid = Number(line.substr(0, 5));
-				const ppid = Number(line.substr(6, 5));
-				let command = line.substr(12, 20).trim();
-				let args = line.substr(33);
-
-				let pos = args.indexOf(command);
-				if (pos >= 0) {
-					pos = pos + command.length;
-					while (pos < args.length) {
-						if (args[pos] === ' ') {
-							break;
-						}
-						pos++;
-					}
-					command = args.substr(0, pos);
-					args = args.substr(pos + 1);
-				}
+			proc = spawn('/bin/ps', [ '-ax', '-o', 'pid:6,ppid:6,comm:20,command' ]);	// we specify the column width explicitly
+			proc.stdout.setEncoding('utf8');
+			proc.stdout.on('data', lines(line => {
```

Comment:
pls keep your change minimum as possible and do not touch irrelevant parts. For example, do not remove `?` from `proc.stdout?.setEncoding` and `proc.stdout?.on`.

Reply:
Thank you for review! 

This is not mine change. These are from newest version of node's debugger file. If you check commit history, you'll see that I have taken new file as is. And only on top of that work applied my own.